src/TableRow.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import NightsCount from "./NightsCount";
+import "./TableRow.css";
+
+function handleClick(highlightColor, setHighlightColor) {
+  if (highlightColor === "red") {
+    setHighlightColor("");
+  } else {
+    setHighlightColor("red");
+  }
+}
+
+const TableRow = (props) => {
+  const [highlightColor, setHighlightColor] = useState("");
+
+  function handleProfileClick(id) {
+    props.setCustomerProfileId(id);
+  }
+
+  return (
+    <tr
+      className={highlightColor}
+      key={props.index}
+      onClick={() => {
+        handleClick(highlightColor, setHighlightColor);
+      }}
+    >
+      <td> {props.aBooking.id} </td>
+      <td> {props.aBooking.title} </td>
+      <td> {props.aBooking.firstName} </td>
+      <td> {props.aBooking.surname} </td>
+      <td> {props.aBooking.email} </td>
+      <td> {props.aBooking.roomId} </td>
+      <td> {props.aBooking.checkInDate} </td>
+      <td> {props.aBooking.checkOutDate} </td>
```

Comment:
you can destructure the props object above to you dont need to type `props.aBooking` on every line